spring-data-vmware-gemfire-server/build.gradle.kts

```diff
@@ -135,24 +133,24 @@ fun getGemFireBaseVersion(): String {
     return "${split[0]}.${split[1]}"
 }
 
-tasks.register("copyJavadocsToBucket") {
-    dependsOn(tasks.named("javadocJar"))
-    doLast {
-        val storage = StorageOptions.newBuilder().setProjectId(project.properties["docsGCSProject"].toString()).build().getService()
-        val blobId = BlobId.of(project.properties["docsGCSBucket"].toString(), "${publishingDetails.artifactName.get()}/${project.version}/${tasks.named("javadocJar").get().outputs.files.singleFile.name}")
```

Comment:
Is this broken or do we not need to do this anymore?

Reply:
I'm thinking we will require a Spring Boot GemFire Javadoc system for this... or maybe we have 2 Javadocs?!